hoomd/md/methods.py

```diff
@@ -132,6 +115,31 @@ def _attach(self):
                                  "")
         super()._attach()
 
+
+    def thermalize_extra_dof(self, seed):
+        """Set the extra degrees of freedom to random values.
+
+        Args:
+            seed (int): Random number seed
+
+        Some integration methods, such as `NVT` and `NPT` employ extra degrees
+        of freedom for the thermostat and/or barostat. `thermalize_extra_dof`
+        sets these to random values. This method performs no operation for
+        integration methods that lack internal degrees of freedom.
+
+        Note:
+            The seed for the pseudorandom number stream includes the
+            simulation timestep and the provided *seed*.
+        """
+
+        if not self._attached:
+            raise RuntimeError(
+                "Call Simulation.operations.schedule before thermalize_extra_dof")
```

Comment:
We should probably change this to `Simulation.run(0)` now.

Reply:
Done.